assets/scss/_custom.scss

```diff
@@ -108,6 +108,10 @@ section {
 
       &.dropdown:hover {
         color: $medium-grey;
+
+        .dropdown-menu {
```

Comment:
Hi @celestehorgan . This change looks good for adding "hovering" to the two drop down menus.
Currently, you have to "click" the version and language drop down menus. This seems to work for all types, except
the individual case studies pages (spotify, ...).
The case studies home page drop down menus work for me. Not sure why the individual case studies pages are special.

To confirm, this change adds hover to the two drop down menus off the main navbar for all pages.
There is something different about the individual case studies pages.

Reply:
Addressed in the latest commits :)